src/GitVersionCore/Helpers/JsonSerializer.cs

```diff
@@ -0,0 +1,280 @@
+using System;
+using System.Collections;
+using System.IO;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using GitVersion.OutputVariables;
+
+namespace GitVersion.Helpers
+{
+    // Credit to https://github.com/neuecc
+    // Inspired by https://gist.github.com/neuecc/7d728cd99d2a1e613362
+    public static class JsonSerializer
```

Comment:
Is this overkill?

Reply:
I can write a simpler one that only supports dictionary in one depth?

Reply:
@asbjornu Please take another look, I chose a more simplified version that can only do objects.

Reply:
Much simpler and sufficient, I think. If we need the full power of a full-fledged JSON serializer, we'll add a dependency to one, but I don't think that need is here at the moment.

Reply:
The problem with adding such dependency is that you may have dependency conflicts with those using the same JSON serializer dependency.
Reason for #2017 😉 When using AppVeyor, that needed ewtonsoft.json dep and the project uses newtonsoft.json in a different version.